perf_test/blas/blas3/KokkosBlas3_gemm_perf_test.hpp

```diff
@@ -127,15 +128,24 @@ typedef struct gemm_args gemm_args_t;
 static std::string gemm_csv_header_str =
     "algorithm,transAtransB,alpha,beta,team_size,vector_len,loop_type,A_dims,B_"
     "dims,C_dims,warm_up_n,"
-    "iter,total_time(s),average_time(s)";
+    "iter,total_time(s),average_time(s),FLOPS,GFLOP/average_time(s)";
 
 /*************************** Internal helper fns **************************/
+// Flop count formula from lapack working note 41: http://www.icl.utk.edu/~mgates3/docs/lawn41.pdf
+static inline int __gemm_flop_count(int a_m, int a_n, int b_k) {
+  return 2 * a_m * b_k * a_n;
```

Comment:
@e10harvey This flop count is for real number, isn't it? Could you please add flop count for complex too?